src/main/java/de/uka/sdq/beagle/core/BlackboardStorer.java

```diff
@@ -0,0 +1,13 @@
+package de.uka.sdq.beagle.core;
+
+import java.io.Serializable;
+
+/**
+ * Interface for classes that wish to write something to the Blackboard.
+ *
+ * @author Joshua Gleitze
+ * @param <WRITTEN_TYPE>
+ *            The type of data the BlackboardStorer wants to write to the blackboard.
+ */
+public interface BlackboardStorer<WRITTEN_TYPE extends Serializable> {
+}
```

Comment:
I didn't get the sense of this interface yet. Each user, that want's to store something special on the blackbord has to create a class (for each object he want's to store!), which implements this empty interface ?!